src/backend/distributed/operations/repair_shards.c

```diff
@@ -127,6 +127,15 @@ bool DeferShardDeleteOnMove = true;
 double DesiredPercentFreeAfterMove = 10;
 bool CheckAvailableSpaceBeforeMove = true;
 
+/*
+ * ShardInterval along with DDL command list to be executed.
+ */
+typedef struct ShardIntervalWithCommandList
```

Comment:
bit long for a simple thing, maybe something like ShardCommandList

Reply:
Updated.